.travis.yml

```diff
@@ -1 +1,12 @@
 language: java
+install: mvn install -DskipTests=true -DfinalName=cassandra-shiro-$TRAVIS_TAG
+script: mvn verify -DfinalName=cassandra-shiro-$TRAVIS_TAG
+deploy:
+  skip_cleanup: true
+  provider: releases
+  api_key:
+    secure: cvh03OXEwdmvmPpohRbM5CChht6sXRfQzBGWDT7kiL/KxEWAY78sphoJQpJwqxKGYOib2CqMQcT6ffoXhl/75VOnXEFFo6oU1eubkMLcoA8FAQuMYyT+TXLC1qkM6yzT58pDBobqTqulA5t0XmPQlyoDIFmvRvI00nrUkv72T3hsevqazQ9PFzVuUIG0LqFbcnOKkC6yXNeY4VkOYd5V1u0HSoFIXnf/ga0E5F7D+lwE0pb5GQ/xX1LdifQukTMUFOgn02zYhbrbrjvATwOuRmDhYme1KkX3pHeuKXKXKfpKJfDUiqFqOHXDFzWZyhfASaqKkNyy/C8jwfGBklaEvA6mpIPvnxu6a7W8ZtZO8Wl0gs+cQST1aT819MVFdX+NgVVTT2QiykKEbKsDehU3Gw5sXOQlgKqAYLmuJhdofXrSQv+euTw0VfIuiPFC4ENr4yZoMuuGlY2sQwgfKJVwIqqTs7DltAyjuaJ9EEYyojMYOewmL+N/IeWfaRVCxMaTp3ofsfKOx7hN2+fEM/KuEzugHGG3PPJWq1BplutB5irEO2Mv5O+AMVPVC9J3bw8scbZOgzOgM2ZAYgxF4eTm4i2kEpjn+ZXrn6EJYZlecwx6aXJ4NFuipGHbDK66wWurAioydWVK5tNadBdABJvd6h5528NMKQnT5wkUFcdTsTU=
```

Comment:
hm, what is this for - it is in the repo for everybody to see, right?

Reply:
it's  encrypted api key for github releases https://docs.travis-ci.com/user/encryption-keys/